include/HL_gio.h

```diff
@@ -73,7 +73,7 @@ typedef struct gio_config_reg
 #define GIO_INTDET_CONFIGVALUE    0U
 #define GIO_POL_CONFIGVALUE	      ((uint32)((uint32)0U << 0U) \
 								 | (uint32)((uint32)0U << 1U) \
-								 | (uint32)((uint32)0U << 2U) \
+								 | (uint32)((uint32)1U << 2U) \
 								 | (uint32)((uint32)0U << 3U) \
 								 | (uint32)((uint32)0U << 4U) \
 								 | (uint32)((uint32)0U << 5U) \

```

Comment:
This would mean more if the bit positions were documented/named. As it is, 0x04 is just as clear and takes 6 less lines :-/